### DISCUSSION TITLE
State of the project

### DISCUSSION THREAD

**Discussion by EnZon3** [EnZon3/Drop#12] [Announcements]
Hey y'all,
As you may have noticed, progress on Drop has stagnated a bit. Don't worry, the project is not dead yet. I've been busy with homework and other priorities, so I haven't really been able to make any extra progress on Drop. This isn't the end though, as spring break is here. I'll be able to make quite a bit of progress on drop during break!

Cheers!

**Comment by MilesWK** [comment]
Oh I hope you will add this and you probably will: Following/Followers.
Glitch doesn't have it and it drives me insane!

**Reply by EnZon3** [reply]
I'll add that of course, it's a planned feature. I'm still trying to figure out the database architecture for it.
